python/setup.py

```diff
@@ -174,6 +174,8 @@ def find_version(*filepath):
     "packaging",
     "pytest",
     "pyyaml",
+    "jsonschema",
+    "importlib_resources",
```

Comment:
BTW, porting the comment from the Autoscaler PR over here: do we need importlib_resources? This is possible to do with just `f.read(os.path.join(ray.__file__, "ray-schema.json"))` right?